index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Mashrou Siha</title>
```

Comment:
Isn't your project called rikaz?

Reply:
Rekaz is the name of the Department that coneys the survey. The project is called Mashrou Siha (Project health).